libethereum/BlockChain.cpp

```diff
@@ -84,17 +84,23 @@ std::ostream& dev::eth::operator<<(std::ostream& _out, BlockChain const& _bc)
 
 ldb::Slice dev::eth::toSlice(h256 const& _h, unsigned _sub)
 {
-#if ALL_COMPILERS_ARE_CPP11_COMPLIANT
```

Comment:
Its the only place in all of the codebase this macro is currently used. And the code appears to be outdated. I'll restore and fix it.